src/lib.rs

```diff
@@ -87,13 +87,19 @@ struct PowerData {
     data: RegionData,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+struct DataVec {
```

Comment:
May I suggest calling this type `NationalData`? It may communicate how it's used and the fact this is the response format of the API endpoints to get national data.